apps/ichat-gw/ichat-gw.config

```diff
@@ -143,4 +143,9 @@ TLSPrivateKey =
 # Pass phrase of private key file (can be left blank)
 TLSPrivateKeyPassPhrase =
 
+########################################################
+# iChat Jabber Connector Settings
+########################################################
 
+# Path to iChat jabber connector (ichat-gw-jc) executable
+IchatJabberConnectorPath = "ichat-gw-jc"
```

Comment:
Ok

On Thu, Aug 25, 2016 at 5:55 PM, Daniel Pocock <notifications@github.com>
wrote:

> In apps/ichat-gw/ichat-gw.config
> <https://github.com/resiprocate/resiprocate/pull/69#discussion_r76293170>:
>
> >
> > +# Path to iChat jabber connector (ichat-gw-jc) executable
> > +IchatJabberConnectorPath = "ichat-gw-jc"
>
> Does it need the double quotes?
>
> Add another line like this example, commented out:
>
> # For Windows:
> #IchatJabberConnectorPath = ichat-gw-jc.exe
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/resiprocate/resiprocate/pull/69/files/9eeee6697add9d8e61a3c28eb9ebf95acd8f0718#r76293170>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIukopAxuNNjccwNOOrrUFVIuKHk3Kh1ks5qjdcogaJpZM4JtVRh>
> .
>



-- 


*Balram PariyarathTech Head | Excel 2016*
Computer Science and Engineering (Batch: 2013-17)
Govt. Model Engineering College, Kochi-21

[image: ☎]:  +91 8129130866
✉: balbbrl@gmail.com <p.balram96@gmail.com>